### PR TITLE
feature(artifact-tests): add tests using centos9

### PR DESCRIPTION
**Pull request by fruch**
now that centos8 is deprecated, we introduce the exact same tests but ontop of centos9

Ref: #7543

### Testing

- [x] 🟢 https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-centos9-test/8/
- [x] 🟢 https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-centos9-web-test/6/
- [x] 🟢   https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-centos9-arm-test/10/
- [x] 🔴  https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-centos9-selinux-test/6/ - https://github.com/scylladb/scylladb/issues/19325
- [x] 🟠  rolling upgrades: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/fruch/job/rolling-upgrade-centos9-test/7/ - running into #7602
- [x] 🟢  nonroot offline: https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-centos9-test/11/

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
now all jobs are working

beside the selinux one, raise an issue:
https://github.com/scylladb/scylladb/issues/19325

@Annamikhlin @yaronkaikov, care to review ?

File: sdcm/mgmt/common.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ def get_distro_name(distro_object):
         Distro.AMAZON2023: "centos8",
         Distro.CENTOS7: "centos7",
         Distro.CENTOS8: "centos8",
+        Distro.CENTOS9: "centos8",
```
Comment (yaronkaikov) [inline sdcm/mgmt/common.py:25]: shouldn't it be centos9?

File: utils/get_supported_scylla_base_versions.py [hunk 2]
```diff
@@ -16,7 +16,8 @@
 # We support to migrate from specific OSS version to enterprise
 supported_src_oss = {'2021.1': '4.3', '2022.1': '5.0', '2022.2': '5.1', '2023.1': '5.2', '2024.1': '5.4'}
 # If new support distro shared repo with others, we need to assign the start support versions. eg: centos8
-start_support_versions = {'centos-8': {'scylla': '4.1', 'enterprise': '2021.1'}}
+start_support_versions = {'centos-8': {'scylla': '4.1', 'enterprise': '2021.1'},
+                          'centos-9': {'scylla': '5.4', 'enterprise': '2024.1'}}
```
Comment (yaronkaikov) [inline utils/get_supported_scylla_base_versions.py:20]: We will have centos9 supported for those releases? i though it will be from 6.0 and above

Comment (fruch) [reply utils/get_supported_scylla_base_versions.py:20]: it's for the upgrade tests, so we can start from those versions

this doesn't state we have anything unique for centos9, and we don't.

File: sdcm/mgmt/common.py (hunk 1 above)
Comment (fruch) [reply sdcm/mgmt/common.py:25]: it's the same, i.e. point to the same RPMs repo url.

**Review by yaronkaikov — APPROVED** [review]

**Comment by fruch** [comment]
@yaronkaikov 

can you add the triggers ? 

also the issue with setup and selinux, it's on your team plate, right ?

**Pull request merged**